va/libva.def

```diff
@@ -83,3 +83,4 @@ EXPORTS
     vaSetSubpictureImage
     vaStatusStr
     vaSyncSurface2
+    vaDisplayIsValid
```

Comment:
LGTM considering this function is part of VAAPI public API:
https://github.com/intel/libva/blob/b75818c092367723c76e716c56d3ef9765ce9e51/va/va.h#L445

Reply:
@dvrogozh thanks for taking a look! Could a maintainer please help merging it? I don't seem to have enough permissions to do it myself.

![image](https://user-images.githubusercontent.com/82116493/216213176-961de1c6-9ab6-45d9-a31a-c6eb1ba2f62a.png)

Reply:
I am just giving a chance for others to review. I will merge it in tomorrow if no concerns from others unless @XinfengZhang won't do that before me.

Reply:
Sounds good thanks :)